client/extensions/woocommerce/woocommerce-services/views/shipping-label/label-purchase-modal/customs-step/style.scss

```diff
@@ -0,0 +1,79 @@
+.customs-step__package-container {
+	margin-bottom: 16px;
+}
+
+.customs-step__package-container:last-child {
+	margin-bottom: 0;
+}
+
+.customs-step__package-name {
+	font-size: 16px;
+	font-weight: 600;
+	margin-bottom: 8px;
+}
+
+.customs-step__restrictions-row {
+	margin-top: 8px;
+	display: flex;
+	flex-wrap: wrap;
+
+	@include breakpoint( '>960px' ) {
+		flex-wrap: nowrap;
+	}
+}
+
+.customs-step__contents-type {
+	@include breakpoint( '>960px' ) {
+		margin-right: 24px;
+	}
+}
+
+.customs-step__contents-type,
+.customs-step__restriction-type {
+	width: 100%;
+
+	@include breakpoint( '>960px' ) {
+		width: 50%;
+	}
+}
+
+.customs-step__item-rows-header {
+	display: none;
+
+	@include breakpoint( '>960px' ) {
+		display: flex;
+	}
+
+	span {
+		font-size: 14px;
+		font-weight: 600;
+		margin-bottom: 8px;
```

Comment:
> Is this intentionally distinct from the 5px under a FormLabel, since it doesn't only apply to the field right below it?

I don't remember if it was intentional, but you're right, if I put `5px` then it looks like it's solely associated with the field below it, instead of being a "header" for multiple rows.

> Not a big deal but these could be FormLabels in any case, if we want to remove repetition in the first two rules.

A `<FormLabel>` needs to be associated with a single `<input>`, and that's not true in this case.